uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/app-documents.js

```diff
@@ -13,6 +13,11 @@ define([
                 $urlRouterProvider.otherwise('/');
 
                 $stateProvider
+                    .resolveForAll({
```

Comment:
Fix the indentation, we're switching to 2 spaces (up to you if you want to fix just this bit or the whole file)